hamocc/mo_ncout_hamocc.F90

```diff
@@ -221,6 +221,9 @@ subroutine ncwrt_bgc(iogrp)
     call ncdims('ks',ks)
     call ncdims('bounds',2)
     call ncdims('time',0)
+    do i=1,nentries
+        call ncputr(param4nc(i)%pname,param4nc(i)%pvalue)
```

Comment:
For the result in the nc-file, it would be maybe nicer to have `call ncputr('Parameter value for '//param4nc(i)%pname,param4nc(i)%pvalue)` or something similar.

Reply:
Hi @JorgSchwinger , this results in ugly formatting of global attributes:
![globa-attr](https://github.com/NorESMhub/BLOM/assets/92309038/dbe1bf54-fedd-41ac-8129-10b1060aff00)
while keeping it simple:
![globa-attr-simple](https://github.com/NorESMhub/BLOM/assets/92309038/2e892619-e991-481c-a86d-5fb8a0ce3d5c)
also eases usage in scripts (output of ncdump - h). Hence, I would stay with the current version.

Reply:
I don't find the 1st one particularly ugly... that's what ncdump always does with blanks. But anyway, the original is ok, too.

Reply:
If you prefer something like `parameter_xxxx`, we can write this during the call to `pinfo_add_entry`.